geomesa-hbase/geomesa-hbase-datastore/src/main/java/org/locationtech/geomesa/hbase/filters/JSimpleFeatureFilter.java

```diff
@@ -104,7 +134,7 @@ public ReturnCode filterKeyValue(Cell v) throws IOException {
     // TODO: Add static method to compute byte array from SFT and Filter.
     @Override
     public byte[] toByteArray() throws IOException {
-        return Bytes.add(getLengthArray(sftString), getLengthArray(filterString));
+        return Bytes.add(Bytes.add(getLengthArray(sftString), getLengthArray(filterString), getLengthArray(transform)), getLengthArray(transformSchema));
```

Comment:
might also be worthwhile to allocate the whole array up front and copy the string bytes in, instead of creating 3 arrays for the strings, then copying them in.